config/default/kustomization.yaml

```
@@ -1,9 +1,8 @@
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 
-# Adds namespace to all resources. Keep it in capm3-system, as it is a
-# dependency for CAPM3
-namespace: capm3-system
+# Adds namespace to all resources
+namespace: ipam-provider-m3-system
```

This is a bit of a mouth full. Could we simplify it a bit?
I suggest `ipam-metal3-system`. For reference the CAPI in-cluster IPAM has the namespace `capi-ipam-in-cluster-system` so following that we would get `capi-ipam-metal3-system`, but since we can use this also without CAPI, I think we can drop the `capi-` prefix.
```suggestion
namespace: ipam-metal3-system
```

---

+1

---

for the in-cluster provider I went with `capi-ipam-in-cluster-`, so it could be `capi-ipam-metal3-` to stay consistent here. I'd argue that it might help to be consistent, so it's easier to navigate for people that are new to CAPI, but it's entirely up to you.